installers/image/bin/pgo-deploy.sh

```diff
@@ -35,9 +35,7 @@ export SERVICE_TYPE=${SERVICE_TYPE:-ClusterIP}
 export METRICS_NAMESPACE=${METRICS_NAMESPACE:-pgo}
 
 # Installer Settings
-export ANSIBLE_CONFIG="/ansible/ansible.cfg"
```

Comment:
Just curious why we moved these out of the script into the Dockerfiles (therefore moving from a single place to set, modify, etc. these vars if need be to two).

Reply:
I noticed that if you exec into the pod without running the script ansible fails because these are not set. I thought that since ansible doesn't work without these being set it would make sense to define the in the image instead of the script. This can definitely be removed considering people should only be running the script but this would get rid of some confusion if anyone is trying to run ansible commands manually.

Reply:
If it helps with development, testing, etc. of then seems like a fair trade-off (also, can't imagine these vars will change often).  Thanks for explanation.